tests/src/Common/PerfHarness/project.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp1.1": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.1.0"
+        },
+        "xunit.performance.api": "1.0.0-alpha-build0049"
+      },
+      "imports": "dnxcore50"
```

Comment:
Do you need this imports statement?  My guess is no.  Also, I take it that coreclr hasn't yet moved to a CLI that is csproj based?